docs/migrate/migrate-estimator.rst

```diff
@@ -6,6 +6,9 @@ The Estimator primitive is used to design an algorithm that calculates expectati
 Background
 ----------
 
+..
+  vale IBMQuantum.Spelling = NO
```

Comment:
"Opflow" was triggering the spellcheck; could be a bug in Vale but can't confirm.